nilearn/plotting/tests/test_js_plotting_utils.py

```diff
@@ -310,6 +315,10 @@ def test_temp_file_removing():
             os.remove(html._temp_file)
         except Exception:
             pass
+        try:
+            os.remove(tmpfile)
+        except Exception:
```

Comment:
Whoa, very nicely done @jeromedockes !
You don't need to add `exception` if you are catching them all.

Reply:
Thanks for letting me know that some exceptions such as KeyboardInterruptException don't derive from the `Exception` class, but would be caught by the `except`.